conda/recipes/gdal/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "GDAL" %}
```

Comment:
I definitely know gdal is in a public channel. Do you need a specific version that's not available in a public channel?

Reply:
Yes.  Unfortunately there is a python dependency that we are using that can't properly parse the version number on geos 3.6.2, because it includes hashes in the version numbers, and since the version of gdal that we need is built with that dependency in all of the public repos we have to build our own.  Those python dependencies will be upgraded soon, and then once we verify that the public versions work with that, we can remove them from here.